composer.json

```diff
@@ -52,6 +51,8 @@
             "Liip\\FunctionalTestBundle\\": ""
         }
     },
+    "minimum-stability": "RC",
```

Comment:
why did you add this?

Reply:
theofidry/alice-data-fixtures is still in RC, so if not provided the user would need to install it himself and the require would not suffice